pkg/api/admin/openshiftcluster.go

```diff
@@ -295,3 +296,7 @@ type SystemData struct {
 	LastModifiedByType CreatedByType `json:"lastModifiedByType,omitempty"`
 	LastModifiedAt     *time.Time    `json:"lastModifiedAt,omitempty"`
 }
+
+type HiveProfile struct {
+	NamespaceName string `json:"namespaceName,omitempty"`
```

Comment:
nit - can we just call this `namespace` as that's how it's typically referenced in k8s code and whatnot.  

```suggestion
	Namespace string `json:"namespace,omitempty"`

Reply:
Updated